### PR TITLE
Angular 12 upgrade

### PR DESCRIPTION
**Pull request by mario-alva**
**What kind of change does this PR introduce?** (check one with "x")

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
```
Comment (Hypercubed) [inline angular.json:3]: Do you know what this is?

**Comment by Hypercubed** [comment]
Think we also need to bump the dep version in projects/swimlane/ngx-datatable/package.json

File: angular.json (hunk 1 above)
Comment (mario-alva) [reply angular.json:3]: The analytics is to share data with the Angular Team. The change occurred after running the upgrade process.

**Comment by mario-alva** [comment]
> Think we also need to bump the dep version in projects/swimlane/ngx-datatable/package.json

So I built the project, and tested it against/in a ng-11 project (the project was Platform) and it was working fine. Do you think this other bump is still necessary? @Hypercubed

**Comment by Hypercubed** [comment]
> > Think we also need to bump the dep version in projects/swimlane/ngx-datatable/package.json
> 
> So I built the project, and tested it against/in a ng-11 project (the project was Platform) and it was working fine. Do you think this other bump is still necessary? @Hypercubed

These (https://github.com/swimlane/ngx-datatable/blob/1a9b6880a98fef535b38b9ff28f0add6c785fffe/projects/swimlane/ngx-datatable/package.json#L6-L8) should be a `>=` or and `||`

**Comment by mario-alva** [comment]
> > > Think we also need to bump the dep version in projects/swimlane/ngx-datatable/package.json
> > 
> > 
> > So I built the project, and tested it against/in a ng-11 project (the project was Platform) and it was working fine. Do you think this other bump is still necessary? @Hypercubed
> 
> These (
> 
> https://github.com/swimlane/ngx-datatable/blob/1a9b6880a98fef535b38b9ff28f0add6c785fffe/projects/swimlane/ngx-datatable/package.json#L6-L8
> 
> ) should be a `>=` or and `||`

Ok, that's done and ready to merge, I believe.

**Review by Hypercubed — APPROVED** [review]

**Comment by maciej-laskowski** [comment]
@Hypercubed when do you plan on releasing new version with Angular 12 support?

**Comment by jdmaguire** [comment]
> @Hypercubed when do you plan on releasing new version with Angular 12 support?

Came to this PR to wonder the same thing. It was in 20.0.0, release September 14th. Just commenting for the next person who comes along; sorry for spamming anyone watching this issue.

**Pull request merged**